### PR TITLE
[zh] Sync architecture picture of chinese #14379

### PR DESCRIPTION
**Pull request by Skeletons430**
## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by windsonsea** [comment]
Do not always try to close, re-open, close, open, close, open, close your PR if any comments come.

Respect comments is to respect reviewers, do you understand?

**Comment by Skeletons430** [comment]
> Do not always try to close, re-open, close, open, close, open, close your PR if any comments come.
> 
> Respecting comments is to respect reviewers, do you understand?

I know.Thanks for teaching

File: content/zh/docs/ops/deployment/vm-architecture/index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Istio 对虚拟机的支持允许将 Kubernetes 集群外的工作负载接入
 
 {{< tabset category-name="network-mode" >}}
 
-{{< tab name="Single-Network" category-value="single" >}}
+{{< tab name="单网络" category-value="single" >}}
```
Comment (windsonsea) [inline content/zh/docs/ops/deployment/vm-architecture/index.md:25]: This file has 3 SVGs:

- single-network.svg
- multi-network.svg
- service-selector.svg

Is any special reason to localize an irrelevant diagram like `discovery.svg`?

**Review by windsonsea — COMMENTED** [review]
This SVG is not included in the markdown file you changed.

File: content/zh/docs/ops/deployment/vm-architecture/index.md (hunk 1 above)
Comment (Skeletons430) [reply content/zh/docs/ops/deployment/vm-architecture/index.md:25]: no

Comment (windsonsea) [reply content/zh/docs/ops/deployment/vm-architecture/index.md:25]: Well, would you like to localize those SVGs included in this markdown file?

- single-network.svg
- multi-network.svg

Seems the previous two diagram can be localized.

Comment (Skeletons430) [reply content/zh/docs/ops/deployment/vm-architecture/index.md:25]: I think I'd like to do it, but it will take some time

Comment (windsonsea) [reply content/zh/docs/ops/deployment/vm-architecture/index.md:25]: All right. Not hurry. You can refer to [SVG localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization/#localize-svg-images).

Comment (Skeletons430) [reply content/zh/docs/ops/deployment/vm-architecture/index.md:25]: Okay, thanks for the direction

**Comment by Skeletons430** [comment]
@windsonsea Hello, if there are no other questions, can you merge this PR first?

**Comment by windsonsea** [comment]
We accept PRs that can add values. However, this PR provides less:

- `discovery.svg` is a useless and redundant image and no any file referenced it in `docs/` folder. It is not necessary to localize this file.
- Two text localized. Yes, it is correct but small.

I have added the comments https://github.com/istio/istio.io/pull/14381#discussion_r1436861945. Seems you need to perform actions below:

- Localize two SVGs mentioned in `content/zh/docs/ops/deployment/vm-architecture/index.md`
- Revert the `discovery.svg`, which is not necessary. This kind of redundant file shall not waste any time of yours and mine.

**Comment by Skeletons430** [comment]
> We accept PRs that can add values. However, this PR provides less:
> 
> * `discovery.svg` is a useless and redundant image and no any file referenced it in `docs/` folder. It is not necessary to localize this file.
> * Two text localized. Yes, it is correct but small.
> 
> I have added the comments [#14381 (comment)](https://github.com/istio/istio.io/pull/14381#discussion_r1436861945). Seems you need to perform actions below:
> 
> * Localize two SVGs mentioned in `content/zh/docs/ops/deployment/vm-architecture/index.md`
> * Revert the `discovery.svg`, which is not necessary. This kind of redundant file shall not waste any time of yours and mine.

Ok, I see

**Comment by windsonsea** [comment]
need to rebase master after you synced your fork:

```
git checkout master
git pull -r
git rebase master
```

**Review by windsonsea — APPROVED** [review]
The redundant svg is deleted. Not sure whether it can be merged or not. Let's have a try.

**Pull request merged**